src-graphql/base64.js

```diff
@@ -1,7 +1,9 @@
 export default function base64(str) {
   if (typeof btoa === 'function') {
     return btoa(str);
-  } else {
+  } else if (typeof Buffer !== 'undefined') { // eslint-disable-line no-negated-condition
```

Comment:
You can test that it's a constructor instead just not being undefined. `typeof Buffer === 'function=`, then you don't need to turn off the lint rule.